Remotion/Core/Core/Context/ISafeContextBoundaryStrategy.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Remotion.Context
+{
+  /// <summary>
+  /// Implements a strategy for <see cref="SafeContextBoundary"/> to restore a previous safe context.
```

Comment:
```suggestion
  /// Implements a strategy for <see cref="SafeContextBoundary"/> to restore a previous context.
```
It's a consistency thing. `SafeContext` as a standalone type isn't an exact match, but reading `safe context` as two words seems odd.